src/background.js

```diff
@@ -21,15 +21,26 @@ function init() {
   browser.alarms.onAlarm.addListener(handleWake);
   browser.notifications.onClicked.addListener(handleNotificationClick);
   browser.runtime.onMessage.addListener(handleMessage);
-  handleWake();
+  browser.storage.local.get().then(items => {
```

Comment:
Not sure if we want to [eventually] add `.catch()` blocks for all the promises too, in case of unexpected errors...

There is an ESLint plugin (naturally), appropriately named [**eslint-plugin-promise**](http://npm.im/eslint-plugin-promise) which makes this pretty easy:

```sh
$ eslint --max-warnings=0 src

/Users/pdehaan/dev/github/mozilla/SnoozeTabs/src/background.js
   24:3  error  Expected catch() or return  promise/catch-or-return
   32:5  error  Expected catch() or return  promise/catch-or-return
  129:5  error  Expected catch() or return  promise/catch-or-return

/Users/pdehaan/dev/github/mozilla/SnoozeTabs/src/popup/snooze-content.js
   32:3  error  Expected catch() or return  promise/catch-or-return
  103:3  error  Expected catch() or return  promise/catch-or-return

✖ 5 problems (5 errors, 0 warnings)
```

```js
  "plugins": [
    "promise",
    "react"
  ],
  "rules": {
    "promise/avoid-new": "warn",
    "promise/catch-or-return": "error",
    "promise/no-callback-in-promise": "warn",
    "promise/no-promise-in-callback": "warn",
    "promise/no-return-wrap": "error",
    "promise/param-names": "error",
    ...
```